humble/ros_packages.txt

```diff
@@ -13,6 +14,10 @@ ros-humble-automotive-platform-msgs
 ros-humble-diagnostic-updater
 ros-humble-gps-msgs
 ros-humble-joy-linux
+ros-humble-lanelet2-core
+ros-humble-lanelet2-io
+ros-humble-lanelet2-projection
+ros-humble-lanelet2-routing
```

Comment:
We have these as a source code in autoware.ai and have made custom changes to them. So I am assuming we can build it using that code once we convert that lanelet2 to `humble`. We previously just checked out this repo in autoware.ai and built there where it found the lanelet2 dependencies from source code. So we should probably remove these for now?

As a separate thought, now I do wonder how diverged are we from the official pacakge. Or whether if it is useful to use updated `ros-humble-lanelet2` package, or at least merge the changes to our existing source code.

Reply:
Ah I see, I'm guessing switching to use the official humble release would require fairly intensive integration testing so it would probably be better to continue to use the autoware.ai packages for now